gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXml.kt

```diff
@@ -188,6 +211,13 @@ open class WriteModuleXml @Inject constructor(_objects: ObjectFactory) : Default
         return modules.toString(PrintOptions(pretty = true, singleLineTextElements = true, useSelfClosingTags = false))
     }
 
+    private fun usemoduleDependencySpecs(): Boolean {
+        if (!requiredIgnitionVersion.isPresent) return false
+
+        val version = requiredIgnitionVersion.get().split(".")
+        return version.size >= 2 && version[0].toInt() >= 8 && version[1].toInt() >= 3
```

Comment:
I haven't tried this yet but there is a bug in the expression. It would not do what we want on, for example, `9.0.0`, `9.0`, or `9`.

```suggestion
        val version = requiredIgnitionVersion.get().split(".").map { it.toInt() }
        if (version[0] >= 8) { return true }
        if (version[0] == 8 && version[1] >= 3) { return true }
        return false
```